app/controllers/users_controller.rb

```diff
@@ -1,14 +1,16 @@
 class UsersController < ApplicationController
+  before_action :logged_in_user, only: %i(index edit update)
+  before_action :correct_user, only: %i(edit update)
+  before_action :admin_user, only: :destroy
+  before_action :load_user, only: %i(show update destroy edit)
```

Comment:
dùng except thay cho only nhé vì except it method hơn

Reply:
Của em có  action nên em để only nhé ạ :D

Reply:
Của em có 7 action nên em để only nhé ạ :D

Reply:
-_- em chuyển về dùng thế nài cũng được ngen, như vậy sẽ ngắn gon hơn đó
```ruby
before_action :load_user, except: %i(new create index)
```